libr/bin/format/pyc/marshal.c

```diff
@@ -182,8 +183,8 @@ static pyc_object *get_long_object(RBuffer *buffer) {
 	char digist2hex[] = "0123456789abcdef";
 
 	st32 ndigits = get_st32 (buffer, &error);
-	if (ndigits < -SIZE32_MAX) {
-		eprintf ("bad marshal data (long size out of range)");
+	if (ndigits < -ST32_MAX) {
```

Comment:
whats the point of this chewck? if ndigits is st32 ..

Reply:
don't ask me. i'm not the one that has written the code.

Reply:
i removed the condition since it was weird.